libs/blockchain-watcher/feature-app/src/lib/helper/stringHelper.ts

```diff
@@ -0,0 +1,25 @@
+import { sha256 } from 'js-sha256'
+
+export class StringHelper {
+  generateDisputeApiSignature(secretKey: string, value: string): string {
+    return sha256.hmac(secretKey, value)
+  }
+
+  generateRandomEthAddr(): string {
```

Comment:
Usually I avoid mixing test-only functionality with application code, but in this case I don't feel too strongly about it.

Reply:
Ah, yes. It's a test-only function. Maybe we can have a `util-test/testHelper.ts` to contain it.

Reply:
It's up to you, just a small thing I noticed.

Reply:
I'll create a new issue to log this since it's not a high priority.